src/cloud/lib/stores/electron.ts

```diff
@@ -20,12 +20,12 @@ import { IpcRendererEvent } from 'electron'
 import { useEffectOnce } from 'react-use'
 import ltSemver from 'semver/functions/lt'
 
-export function addToWebContents(eventName: string, callback: any) {
-  ;(window as any).__ELECTRON_ONLY__.addWebContentsEvent(eventName, callback)
+export function addFoundInPageListener(callback: any) {
```

Comment:
Please provide a proper type for `callback`. Should be `(matches: number | null) => void`

Reply:
Done